filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -1843,11 +1842,11 @@ void OpenGLDriver::setMinMaxLevels(Handle<HwTexture> th, uint32_t minLevel, uint
     // Must fit within int8_t.
     assert_invariant(minLevel <= 0x7f && maxLevel <= 0x7f);
 
-    t->gl.baseLevel = minLevel;
+    /*t->gl.baseLevel = minLevel;
     glTexParameteri(t->gl.target, GL_TEXTURE_BASE_LEVEL, t->gl.baseLevel);
 
     t->gl.maxLevel = maxLevel; // NOTE: according to the GL spec, the default value of this 1000
-    glTexParameteri(t->gl.target, GL_TEXTURE_MAX_LEVEL, t->gl.maxLevel);
+    glTexParameteri(t->gl.target, GL_TEXTURE_MAX_LEVEL, t->gl.maxLevel);*/
```

Comment:
Add a note to the PR desc: Fixing a perf. issue (CPU copies inside ANGLE). [GFX-1583](https://shapr3d.atlassian.net/browse/GFX-1583) tracks this.

Reply:
Added comment in the description.